submissions/description/application_level_denial_of_service_dos/high_impact_and_or_medium_difficulty/template.md

```diff
@@ -0,0 +1,30 @@
+# Application-Level Denial of Service: High Impact or Medium Difficulty
+
+## Overview of the Vulnerability
+
+Application-level denial of service (DoS) attacks are designed to deny service to users of an application by flooding it with many HTTP requests. This makes it impossible for the server to respond to legitimate requests in any practical time frame.
+
+There is an application-level DoS vulnerability within this application that has high impact or medium difficulty to be performed. An attacker can use this vulnerability to exhaust resources, making the application unavailable for its designed purpose to legitimate users.
```

Comment:
Clarifying impact here will help us understand why this is a high and not critical impact issue

> An attacker can use this vulnerability to exhaust resources, making the application unavailable for its designed purpose to legitimate users, but not take down the application for all users.